.github/workflows/crowdin-upload.yml

```diff
@@ -48,8 +48,9 @@ jobs:
           upload_sources_args: --preserve-hierarchy
           upload_translations_args: --preserve-hierarchy
           download_translations: false
-          auto_approve_imported: true
+          auto_approve_imported: false
```

Comment:
Sorry if you've explained this already, but will we need to approve translations added directly to the JSON files on Crowdin? I'm still not seeing the whole picture. 🙇‍♀️

Reply:
For the crowdin version we are testing the auto approve may not work as I expect it. But what I hope is that all that are already approved in Crowdin remain approved in the new version branch. (this feature is mostly useful for uploading new "pre-approved" translations.

For adding a translation directly to JSON files, what I hope is that from now on any new translations are added in Crowdin and not github -- except english. Otherwise any translation directly added will go into CrowdIN and need approval. But the approval will be not meaningful since the approval is a requirement for download and so git diff in this case will not be visible.

If we add "missing" those will upload and the CrowdIn language owner will delegate/translate/approve.

Reply:
I think I better understand it now, thank you!